mission_control/navigator_launch/launch/cameras.launch

```diff
@@ -0,0 +1,80 @@
+<launch>
+  <!-- This is the default pointgrey launch file with our id's and our names  -->
+
+  <arg name="front_camera_serial" default="15223362" />
+  <arg name="front_camera_calibrated" default="0" />
+  <arg name="down_camera_serial" default="10282860" />
+  <arg name="down_camera_calibrated" default="0" />
```

Comment:
We'll get rid of them once I get to Gainesville and verify there's not something screwy happening with the cameras. I set up the cameras/launches forever ago so I might have forgotten something, or someone might have changed something.